forge-gui/res/cardsfolder/upcoming/tribute_to_urborg.txt

```diff
@@ -0,0 +1,10 @@
+Name:Tribute to Urborg
+ManaCost:1 B
+Types:Instant
+K:Kicker:1 U
+A:SP$ Pump | ValidTgts$ Creature | IsCurse$ True | NumAtt$ -2 | NumDef$ -2 | SubAbility$ KickerPump | SpellDescription$ Target creature gets -2/-2 until end of turn. If this spell was kicked, that creature gets an additional -1/-1 until end of turn for each instant and sorcery card in your graveyard.
+SVar:KickerPump:DB$ Pump | Condition$ Kicked | Defined$ Targeted | NumAtt$ -X | NumDef$ -X
```

Comment:
For StackDescription, I'd do:
Line 5: `SpellDescription$ Target creature gets -2/-2 until end of turn.` (remove the rest)
Line 6: `StackDescription$ SpellDescription | SpellDescription$ If this spell was kicked, that creature gets an additional -1/-1 until end of turn for each instant and sorcery card in your graveyard.`